presentation/src/main/java/com/example/myapplication/MainActivity.kt

```diff
@@ -3,44 +3,89 @@ package com.example.myapplication
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Surface
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import com.example.myapplication.ui.theme.MyApplicationTheme
+import com.example.myapplication.ui.theme.Pink
+import com.example.myapplication.ui.theme.White
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             MyApplicationTheme {
-                // A surface container using the 'background' color from the theme
                 Surface(
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colorScheme.background
                 ) {
-                    Greeting("Android")
+                    TestFrame()
                 }
             }
         }
     }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "0일차 과제~!",
-        modifier = modifier
-    )
+fun TestFrame() {
+    Column(
+        modifier = Modifier
+            .background(White)
+            .fillMaxSize()
+            .padding(start = 20.dp, end = 20.dp, top = 35.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Spacer(
+            modifier = Modifier
+                .background(Pink)
+                .fillMaxWidth()
+                .height(103.dp)
+        )
+        Spacer(Modifier.height(26.dp))
```

Comment:
```suggestion
        Spacer(modifier = Modifier.height(26.dp))
```
코틀린의 `Named Argument`를 이용해서 쓰면 더 좋을것 같아요! 코드 컨벤션은 아직 정해지는 않았지만 주로 사용하는것이 읽을때 이로울수 있어요!